.blackfire.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
```

Comment:
With the "See" reference, my first question is where do I find the COPYING.txt file. If you keep this sentence, consider adding the link reference.

Alternatively: 

Since the copying.txt file links to the license.txt file,  maybe you can just replace "See COPYING.txt for license details with this sentence:  

This file is licensed under the [OSL-3.0 license](http://opensource.org/licenses/osl-3.0.php).

Reply:
It's in root of repo. This is a copy-paste from reviewing team, which was requested to be added, so I doubt we can change it sadly

Reply:
I know it's in the root of the repo, but I shouldn't have to ask the question ;-)

Reply:
Based on these examples, this looks nice:

```
# Copyright © Magento, Inc. All rights reserved.
# Licensed under the OSL 3.0 License. See License.txt in the project root for license information.
```

Reply:
Reverted changes